tests/xlsx_export.test.ts

```diff
@@ -958,6 +959,60 @@ describe("Test XLSX export", () => {
     });
   });
 
+  describe("Images", () => {
+    const imageData = {
```

Comment:
it must be wrapped inside a `beforeEach` function or returned by a function called each time.
We don't have any guarantee the object is not modified by one of the test (now or in the future) and tests would no longer be independent from one another (hello non-determinism :wave: )